### DISCUSSION TITLE
Is it fully compatible with Apache Doris?

### DISCUSSION THREAD

**Discussion by MuXueMaster** [mysql-net/MySqlConnector#1430] [Q&A]
I want to use MySqlConnector to connect to the Apache Doris database for operations such as adding, modifying, deleting, and querying. Can MySqlConnector meet these needs?

 the Url Of Apache Doris 
https://doris.apache.org/

**Comment by bgrainger** [comment]
I haven't ever tried MySqlConnector with Doris. 

Have you? If so, what happened? Did it work?

**Reply by bgrainger** [reply]
I couldn't find an easy way to run it with Docker (and I wasn't interested in building my own custom FE and BE images just to test it).

**Reply by MuXueMaster** [reply]
I plan to use Doris as a storage database to store business data and provide query functionality.

**Reply by MuXueMaster** [reply]
Apache Doris is becoming increasingly popular. It is compatible with the MySQL 5.7 protocol
